assets/src/block-styles/index.js

```diff
@@ -0,0 +1,83 @@
+
+const {unregisterBlockStyle, registerBlockStyle} = wp.blocks;
+const {select} = wp.data;
+const {__} = wp.i18n;
+
+export const registerBlockStyles = () => {
+  // Remove Take Action and Campaign covers styles for Covers block in campaigns
```

Comment:
I'm not sure we'll be moving the Covers block to the theme (potentially replacing it with the Actions List), so maybe we shouldn't move this code?

Reply:
Make sense for me!